### PR TITLE
Sql lesson

### PR DESCRIPTION
**Pull request by inlibra-cpa**
CRUD with SQL. Lesson 1

### CODE REVIEW

File: sql.txt [hunk 1]
```diff
@@ -1,14 +1,20 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
 Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
-
+SELECT CustomerName, ContactName, Country FROM Customers ORDER BY Country LIMIT 10 OFFSET 10;
```
Comment (ZipDevil) [inline sql.txt:3]: 👍

File: sql.txt [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
 (2) Select all columns from the Customer table where the ContactName starts with A.  Paste your SQL statement below.
-
+SELECT * FROM Customers WHERE ContactName LIKE "A%";
```
Comment (ZipDevil) [inline sql.txt:5]: 👍

File: sql.txt [hunk 3]
```diff
@@ -1,14 +1,20 @@
...
 (3) Select all columns from the OrderDetails table where the ProductID is 51 and the quantity is greater than 10.
 Paste your SQL statement below.
-
+SELECT * FROM OrderDetails WHERE ProductID = 51 AND Quantity > 10;
```
Comment (ZipDevil) [inline sql.txt:8]: 👍

File: sql.txt [hunk 4]
```diff
@@ -1,14 +1,20 @@
...
 (4) Insert 3 rows into the Products table.  Note that you will have to specify a valid SupplierID and CategoryID, 
 corresponding to rows from the Supplier and Category tables.  Paste your three SQL statements below.
+INSERT INTO Products (SupplierID, CategoryID)
+VALUES
+      (27, 8),
+      (28, 8),
+      (29, 8);
```
Comment (ZipDevil) [inline sql.txt:15]: Well done.

File: sql.txt [hunk 5]
```diff
@@ -1,14 +1,20 @@
...
 
 (5) Update the two top rows of the Products Table to increase the price by 1.50. (Get SQL to do the addition for you.)  Paste your SQL statement below.
-
+UPDATE Products SET price = (price+1.5) WHERE ProductID <= 2;
```
Comment (ZipDevil) [inline sql.txt:18]: 👍

File: sql.txt [hunk 6]
```diff
@@ -1,14 +1,20 @@
...
 (6) Delete all rows of the Products Table where the price is less than 7.00. Paste your SQL statement below.
+DELETE FROM products WHERE price < 7;
```
Comment (ZipDevil) [inline sql.txt:20]: 👍

**Review by ZipDevil — COMMENTED** [review]
Great job on making short and clean SQL queries!

**Comment by inlibra-cpa** [comment]
Thank you Jasper for your review and encouraging feedback!:-)

**Pull request closed without merging**